### PR TITLE
Add K10 chart version 4.5.6

### PR DESCRIPTION
**Pull request by akankshakumari393**
This PR is to add K10 chart version 4.5.6 on to rancher Marketplace.

### CODE REVIEW

File: packages/k10/generated-changes/overlay/app-readme.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+The K10 data management platform, purpose-built for Kubernetes, provides enterprise operations teams an easy-to-use, scalable, and secure system for backup/restore, disaster recovery, and mobility of Kubernetes applications.
+
+K10’s application-centric approach and deep integrations with relational and NoSQL databases, Kubernetes distributions, and all clouds provides teams the freedom of infrastructure choice without sacrificing operational simplicity. Policy-driven and extensible, K10 provides a native Kubernetes API and includes features such full-spectrum consistency, database integrations, automatic application discovery, multi-cloud mobility, and a powerful web-based user interface.
+
+For more information, refer the docs [https://docs.kasten.io/](https://docs.kasten.io/)
```
Comment (akankshakumari393) [inline packages/k10/generated-changes/overlay/app-readme.md]: the content in this file provides the description of k10 Application.

File: packages/k10/generated-changes/overlay/questions.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  
+### Service Account
+
+- variable: serviceAccount.name
+  description: "Name of a service account in the target namespace that has cluster-admin permissions. This is needed for the K10 to be able protect cluster resources."
+  type: string
+  label: Service Account Name
+  required: false
+  default: ""
+  group: "Service Account"
```
Comment (akankshakumari393) [inline packages/k10/generated-changes/overlay/questions.yml:44]: the above file specifies the helm flag that we would be allowing users to customize while installing the chart.

File: packages/k10/package.yaml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+url: https://charts.kasten.io/k10-4.5.6.tgz
+version: 4.5.6
```
Comment (akankshakumari393) [inline packages/k10/package.yaml]: this provides the chart URL to use and prepare the charts and other related files.

File: packages/k10/generated-changes/patch/Chart.yaml.patch [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+@@ -8,3 +8,8 @@
+   name: kastenIO
+ name: k10
+ version: 4.5.6
++kubeVersion: ">= 1.17"
++annotations:
++  catalog.cattle.io/certified: partner
++  catalog.cattle.io/release-name: k10
++  catalog.cattle.io/display-name: kasten/k10 
+\ No newline at end of file
```
Comment (akankshakumari393) [inline packages/k10/generated-changes/patch/Chart.yaml.patch]: chart.yaml file with additional mandatory configuration for rancher.

File: charts/k10/k10/4.5.6/app-readme.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+The K10 data management platform, purpose-built for Kubernetes, provides enterprise operations teams an easy-to-use, scalable, and secure system for backup/restore, disaster recovery, and mobility of Kubernetes applications.
+
+K10’s application-centric approach and deep integrations with relational and NoSQL databases, Kubernetes distributions, and all clouds provides teams the freedom of infrastructure choice without sacrificing operational simplicity. Policy-driven and extensible, K10 provides a native Kubernetes API and includes features such full-spectrum consistency, database integrations, automatic application discovery, multi-cloud mobility, and a powerful web-based user interface.
```
Comment (pavannd1) [inline charts/k10/k10/4.5.6/app-readme.md]:
```suggestion
K10’s application-centric approach and deep integrations with relational and NoSQL databases, Kubernetes distributions, and all clouds, provide teams the freedom of infrastructure choice without sacrificing operational simplicity. Policy-driven and extensible, K10 provides a native Kubernetes API and includes features such as full-spectrum consistency, database integrations, automatic application discovery, multi-cloud mobility, and a powerful web-based user interface.
```

File: packages/k10/generated-changes/overlay/app-readme.md [hunk 6]
```diff
@@ -0,0 +1,5 @@
+The K10 data management platform, purpose-built for Kubernetes, provides enterprise operations teams an easy-to-use, scalable, and secure system for backup/restore, disaster recovery, and mobility of Kubernetes applications.
+
+K10’s application-centric approach and deep integrations with relational and NoSQL databases, Kubernetes distributions, and all clouds provides teams the freedom of infrastructure choice without sacrificing operational simplicity. Policy-driven and extensible, K10 provides a native Kubernetes API and includes features such full-spectrum consistency, database integrations, automatic application discovery, multi-cloud mobility, and a powerful web-based user interface.
```
Comment (pavannd1) [inline packages/k10/generated-changes/overlay/app-readme.md]:
```suggestion
K10’s application-centric approach and deep integrations with relational and NoSQL databases, Kubernetes distributions, and all clouds, provide teams the freedom of infrastructure choice without sacrificing operational simplicity. Policy-driven and extensible, K10 provides a native Kubernetes API and includes features such as full-spectrum consistency, database integrations, automatic application discovery, multi-cloud mobility, and a powerful web-based user interface.
```

File: packages/k10/package.yaml (hunk 3 above)
Comment (pavannd1) [inline packages/k10/package.yaml]:
```suggestion
version: 4.5.6

```

File: packages/k10/generated-changes/patch/Chart.yaml.patch (hunk 4 above)
Comment (pavannd1) [inline packages/k10/generated-changes/patch/Chart.yaml.patch]:
```suggestion
\ No newline at end of file
```
Should this be present? How was this generated?

File: charts/k10/k10/4.5.6/files/kasten-logo.svg [hunk 7]
```diff
@@ -0,0 +1,24 @@
+<svg viewBox="0 0 986 215" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (pavannd1) [inline charts/k10/k10/4.5.6/files/kasten-logo.svg:1]: Where will this logo be displayed? 
Is this the one with `Kasten By Veeam` or just `Kasten`? Is there a favicon with `By Veeam` available?

File: charts/k10/k10/4.5.6/Chart.yaml [hunk 8]
```diff
@@ -0,0 +1,15 @@
+annotations:
+  catalog.cattle.io/certified: partner
+  catalog.cattle.io/display-name: kasten/k10
+  catalog.cattle.io/release-name: k10
+apiVersion: v2
+appVersion: 4.5.6
+description: Kasten’s K10 Data Management Platform
+home: https://kasten.io/
+icon: https://docs.kasten.io/_static/kasten.png
```
Comment (akankshakumari393) [inline charts/k10/k10/4.5.6/Chart.yaml]: Icon URL Information is provided in the `Chart.yaml`. Currently, we are using [https://docs.kasten.io/_static/kasten.png](https://docs.kasten.io/_static/kasten.png). It has only `Kasten`. I will update the PR to use [https://docs.kasten.io/_static/kasten-logo-vertical.png](https://docs.kasten.io/_static/kasten-logo-vertical.png). This one uses `Kasten by Veaam`.

File: charts/k10/k10/4.5.6/files/kasten-logo.svg (hunk 7 above)
Comment (akankshakumari393) [reply charts/k10/k10/4.5.6/files/kasten-logo.svg:1]: this logo is displayed on the 
1. Charts page on Marketplace 
image


2. On Application's description page 

image


 And this information is loaded from Icon field. I will update it to use `kasten by veaam` See my comments here [https://github.com/kastenhq/partner-charts/pull/1#discussion_r783650784](https://github.com/kastenhq/partner-charts/pull/1#discussion_r783650784)

File: packages/k10/generated-changes/patch/Chart.yaml.patch (hunk 4 above)
Comment (akankshakumari393) [reply packages/k10/generated-changes/patch/Chart.yaml.patch]: removed that. I did not add a new line to the original file which was used to generate this.

File: charts/k10/k10/4.5.6/app-readme.md [hunk 9]
```diff
@@ -0,0 +1,5 @@
+The K10 data management platform, purpose-built for Kubernetes, provides enterprise operations teams an easy-to-use, scalable, and secure system for backup/restore, disaster recovery, and mobility of Kubernetes applications.
+
+K10’s application-centric approach and deep integrations with relational and NoSQL databases, Kubernetes distributions, and all clouds provide teams the freedom of infrastructure choice without sacrificing operational simplicity. Policy-driven and extensible, K10 provides a native Kubernetes API and includes features such as full-spectrum consistency, database integrations, automatic application discovery, multi-cloud mobility, and a powerful web-based user interface.
+
+For more information, refer the docs [https://docs.kasten.io/](https://docs.kasten.io/)
```
Comment (pavannd1) [inline charts/k10/k10/4.5.6/app-readme.md]:
```suggestion
For more information, refer to the docs [https://docs.kasten.io/](https://docs.kasten.io/)
```

**Review by pavannd1 — COMMENTED** [review]
LGTM. Ready for upstream 👍🏼

**Comment by akankshakumari393** [comment]
Closing This PR as it was created for testing K10 chart on Rancher Marketplace

**Pull request closed without merging**